### DISCUSSION TITLE
Unable to login to admin interface

### DISCUSSION THREAD

**Discussion by CitizenBeta** [dani-garcia/vaultwarden#4749] [Q&A]
After updating the vaultwarden to v1.31.0, I cannot open the admin interface any more. I've tried to edit the config ADMIN_TOKEN=****, but it's not working.

**Comment by BlackDex** [comment]
What did you try to do and check?
 - What URL did you try to access
 - What message did you get
 - What did the Vaultwarden logs report
 - What does you reverse proxy report
